wgsl/index.bs

```diff
@@ -2082,8 +2082,20 @@ shift_expression
   | shift_expression SHIFT_RIGHT additive_expression
         OpShiftRightLogical or OpShiftRightArithmetic
 
-relational_expression
+and_expression
   : shift_expression
+  | and_expression AND equality_expression
```

Comment:
This one needs to be updated as well

Reply:
@dj2 Does the latest commit address concerns?